server/ServerApp.cpp

```diff
@@ -3322,17 +3322,28 @@ void ServerApp::PostCombatProcessTurns() {
 
 void ServerApp::CheckForEmpireElimination() {
     std::set<Empire*> surviving_empires;
+    std::set<Empire*> surviving_human_empires;
```

Comment:
Prefer storing empire ids rather than pointers

Reply:
Ok, but it was used to call `Win` on found empires.